File: backend/src/main/java/org/blogsphere/blog/Payload/Response/PostResponse.java
package org.blogsphere.blog.Payload.Response;

import java.time.LocalDateTime;
import java.util.Set;

public record PostResponse(
    String id,

    String title,

    String content,

    String coverImage,

    LocalDateTime createdAt,

    LocalDateTime updatedAt,

    LocalDateTime publishedAt,

    boolean isPublished,

    Set<String> tagsId,

    //Set<String> commentsId,

    String createdBy
) {
    
}

File: frontend/src/app/models/post-response.ts
export interface PostResponse {
    id: String;
    title: String;
    content: String;
    coverImage: String;
    createdAt: Date;
    updatedAt: Date;
    publishedAt: Date;
    isPublished: Boolean;
    tagsId: String[];
    commentsId: String[];
    createdById: String;

}
